recipes/tardis/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "tardis-sn" %}
+{% set version = "3.0.a1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  git_url: https://github.com/tardis-sn/tardis.git
+  git_branch: master
+
+build:
+  number: 0
+  skip: True   # [win or py != 36]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    
+  host:
+    - python
+    - pip
+    - setuptools
+    - astropy =3
+    - Cython =0.29
+
+  run:
+    - python
+    - astropy =3
+    - {{ pin_compatible('numpy') }}    
```

Comment:
for now we need numpy=1.15 (for pandas.to_hdf)

Reply:
Does it need to be exactly `1.15` instead of `>=1.15`? Numpy is exceptionally good about backwards compatibility.

Also does tardis build against the numpy C API? If so, read [this](http://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy).